config/configauth/README.md

```diff
@@ -1,17 +1,25 @@
 # Authentication configuration for receivers
 
-This module allows server types, such as gRPC and HTTP, to be configured to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator. Currently, only bearer token authentication is supported, although the module is ready to accept new authenticators.
+This module allows server types, such as gRPC and HTTP, to be configured to perform authentication for requests and/or RPCs. Each server type is responsible for getting the request/RPC metadata and passing down to the authenticator.
```

Comment:
We'll need to expand the docs to explain how to add new extensions

Reply:
Let me know what you think of the addition, I'm having a tough time writing this part for some reason